serpent-moltres/isotope-distribution/README.md

```diff
@@ -0,0 +1,17 @@
+
+This study compares the homogenized cross-sections from:
```

Comment:
Maybe add some text describing what the comparison entails (what compare.py outputs). Figures or values?

Reply:
done